pkg/network/vmicontroller/status.go

```diff
@@ -124,3 +127,20 @@ func calculateSecondaryIfaceStatuses(
 
 	return interfaceStatuses, nil
 }
+
+func filterUnspecifiedIfaces(
```

Comment:
nit: It is unclear to what "unspecified" refers to. Maybe `filterUnspecifiedSpecIfaces`.

Reply:
Done.